### ISSUE TITLE
HasDropDown: should set aria-expanded initially

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#441]
When you open a dropdown, `aria-expanded` is set to `"true"`, and when you close the dropdown, `aria-expanded` is set to `"false"`.  However, it's not set at all initially.  It should be initially set to `"false"`.
